engine/report/json/report_json.cpp

```diff
@@ -1022,6 +1022,24 @@ void profileset_json2( const profileset::profilesets_t& profileset, const sim_t&
 
                      obj[ "iterations" ] = as<uint64_t>( result.iterations() );
 
+                     if ( sim.profileset_cull.enabled )
+                     {
+                       obj[ "profileset_cull_best" ] = ( sim.profileset_cull.best_name == profileset->name() );
+                       if ( sim.profileset_cull.best_name == profileset->name() )
+                       {
+                         obj[ "profileset_cull_best_error" ] = sim.profileset_cull.best_error;
+                       }
```

Comment:
Perhaps I'm missing something, but is there a good reason to keep this information redundant? I can't think of a use case that wouldn't be covered by just including the best name/error in the JSON of the parent (non-profileset) sim.

Reply:
Nope - not missing something. Just cruft. Thanks - fixed.